spot-electron/package.json

```diff
@@ -14,13 +24,17 @@
         "devtron": "1.4.0",
         "electron": "5.0.3",
         "electron-builder": "20.43.0",
+        "electron-notarize": "0.1.1",
         "electron-rebuild": "1.8.5",
         "eslint": "5.16.0",
         "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi#1.0.1",
         "eslint-plugin-flowtype": "3.10.1",
         "eslint-plugin-import": "2.17.3",
         "eslint-plugin-jsdoc": "7.2.3"
     },
+    "dmg": {
+        "sign": false
```

Comment:
Why false?

Reply:
With the extra notarization signing step after build the signing happens there and not during regular build phase.